cli/src/commands.rs

```diff
@@ -50,9 +50,12 @@ pub(crate) enum Command {
         /// Amount that will be received, in satoshi
         receiver_amount_sat: u64,
 
-        // The optional fee rate to use, in satoshi/vbyte
+        /// The optional fee rate to use, in satoshi/vbyte
         #[clap(short = 'f', long = "fee_rate")]
         sat_per_vbyte: Option<u32>,
+
+        #[arg(short, long)]
```

Comment:
Can you add a docs comment?

Reply:
Done: 5df694a0eef34fb8e38e9efedee93ba78cad35cd